test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/ContactDetailsTooManyAttemptsPasscodeControllerSpec.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.controllers
+
+import base.unit.ControllerSpec
+import org.mockito.ArgumentMatchers.any
+import org.mockito.BDDMockito.`given`
+import org.mockito.Mockito.reset
+import org.scalatest.matchers.must.Matchers.convertToAnyMustWrapper
+import play.api.http.Status.OK
+import play.api.test.Helpers.status
+import play.twirl.api.HtmlFormat
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.too_many_attempts_passcode_page
+import uk.gov.hmrc.play.bootstrap.tools.Stubs.stubMessagesControllerComponents
+
+class ContactDetailsTooManyAttemptsPasscodeControllerSpec extends ControllerSpec {
+  private val page = mock[too_many_attempts_passcode_page]
+  private val mcc  = stubMessagesControllerComponents()
+
+  private val controller =
+    new ContactDetailsTooManyAttemptsPasscodeController(mcc = mcc, page = page)
+
+  override protected def beforeEach(): Unit = {
+    super.beforeEach()
+    val registration = aRegistration()
+    mockRegistrationFind(registration)
+    given(page.apply()(any(), any())).willReturn(HtmlFormat.empty)
+  }
+
+  override protected def afterEach(): Unit = {
+    reset(page)
+    super.afterEach()
+  }
+
+  "Too many attempts " should {
+
+    "page is displayed " when {
+
+      "display page method is invoked" in {
```

Comment:
I think we should construct these so that they read properly when run via sbt which includes the words `should` and `when`. So, something like:
```
"The 'too many attempts' page" should {
  "be displayed" when {
    "the display page method is invoked" in {
      ...
    }
  }
}
```

Reply:
Corrected